pkgs/server/src/modules/nameProfession/entity.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  Entity,
+  Column,
+  CreateDateColumn,
+  UpdateDateColumn,
+  ManyToOne
+} from 'typeorm';
+
+import { Name } from '../names';
+
+export enum Profession {
+    Actor = 'ACTOR',
+    Actress = 'ACTRESS',
+    Producer = 'PRODUCER',
+    Director = 'DIRECTOR',
+    Miscellaneous = 'MISCELLANEOUS'
+  }
+
+@Entity({
+  name: 'nameProfession',
```

Comment:
Can we use snake case for Table names please?

```suggestion
  name: 'name_profession',
```